description/dvd.rdfs.ttl

```diff
@@ -0,0 +1,39 @@
+@prefix haDVD: <https://data.hetarchief.be/ns/dvd/> .
```

Comment:
It could be nice to have everything in one and the same model/namespace, although putting everything into separate models also has the current advantage that we can display it as (sub)models on our developer documentation?

Reply:
> It could be nice to have everything in one and the same model/namespace, although putting everything into separate models also has the current advantage that we can display it as (sub)models on our developer documentation?

Is that not still possible since they are different shacl files?

> In any case, I was wondering whether we really have to set up a whole model for e.g. DVD when we have so little stuff to model. It seems a bit overkill and perhaps you could throw DVD and AV together, but that might become messy later on.

That's a good suggestion actually! We can always split them again if they ever become too big.

Reply:
I'll add the AV model to the same file then.

Perhaps @lennertvandevelde can chime in on the namespace question. I don't know how SHACL2MD would treat them if they share the same namespace, but are in different files?

Reply:
I've added all of the descriptive ontologies to haDes.